bitrise.yml

```diff
@@ -377,6 +377,12 @@ workflows:
         - test_repetition_mode: retry_on_failure
         - maximum_test_repetitions: "5"
         - scheme: PaymentSheet Example
+    - xcode-test@4:
```

Comment:
this adds the UI test to be run on nightly builds; we may or may not want this but should be OK

I still will be exploring running this more often, and having a Financial Connections specific trigger, but that's a separate issue